TeamCode/src/main/java/org/firstinspires/ftc/teamcode/utilities/Arm.java

```diff
@@ -38,9 +38,7 @@ public void toScoreBucketPos() {
         arm.setPosition(0.25);
     }
 
-    /* public void toLowScoreBucketPos() {
-        arm.setPosition(0.25);
-    }*/
+    public void toLowScoreBucketPos() { arm.setPosition(0.25); }
```

Comment:
lgtm - did u tune the position using the robot?

Reply:
> lgtm - did u tune the position using the robot?

Nope